docs/ledger_integration_test.md

```diff
@@ -26,13 +26,13 @@ The Ledger Signer's integration tests are disabled by default because they requi
 4. If you only want to run Ledger integration tests, use this command: `npm run test ledger-signer.spec.ts`
 5. As the tests include operation like transfer of token and contract origination, the tests will take some time to complete. You will be prompt on the Ledger to confirm the operations.
 
-## Falling tests
+## Failing tests
 
 There is also a set of integration tests used to verify the behavior when the user declines the Ledger's prompt.
 
 To run these tests, you need to:
 
 1. Open `Tezos Wallet app` on your Ledger device.
-2. Remove `./ledger-signer-falling-tests.spec.ts` from `"testPathIgnorePatterns"` in the package.json.
-3. If you only want to run these tests, use this command: `npm run test ledger-signer-falling-tests.spec.ts`
-4. You will need to decline all Ledger prompts.
\ No newline at end of file
+2. Remove `./ledger-signer-failing-tests.spec.ts` from `"testPathIgnorePatterns"` in the package.json.
```

Comment:
@mkernaghan Can you also fix the filename `taquito/integration-tests/ledger-signer-falling-tests.spec.ts` as part of this PR?